container_env_files/ui.env

```diff
@@ -0,0 +1,29 @@
+# Define environment
+REACT_APP_ENV=production
+# For all environments
+REACT_APP_GATEWAY_URL=http://localhost:${REVERSE_PROXY_PORT}/cgw
+REACT_APP_GOOGLE_ANALYTICS=
+REACT_APP_INFURA_TOKEN=
+REACT_APP_IPFS_GATEWAY=https://ipfs.io/ipfs
+REACT_APP_SENTRY_DSN=
+REACT_APP_SAFE_APPS_RPC_INFURA_TOKEN=
+REACT_APP_GNOSIS_APPS_URL=
+
+# For production environments
+REACT_APP_INTERCOM_ID=
+REACT_APP_PORTIS_ID=
+REACT_APP_SQUARELINK_ID=
+REACT_APP_FORTMATIC_KEY=
+REACT_APP_COLLECTIBLES_SOURCE=
+REACT_APP_ETHERSCAN_API_KEY=
+REACT_APP_ETHGASSTATION_API_KEY=
+
+# Versions
+REACT_APP_LATEST_SAFE_VERSION=
+
+# Leave it untouched, version will set using dotenv-expand
+REACT_APP_APP_VERSION=$npm_package_version
+
+# Contracts Addresses
+REACT_APP_SPENDING_LIMIT_MODULE_ADDRESS=0x9e9Bf12b5a66c0f0A7435835e0365477E121B110
```

Comment:
Should this be set by default :thinking:

Reply:
I copied it from https://github.com/gnosis/safe-react/blob/dev/.env.example , I didn't want to break anything (cc @gnosis/safe-web)

Reply:
We can definitely remove this from `.env.example`.

The spending limit module address we use (when not in Secrets) is [here](https://github.com/gnosis/safe-react/blob/dev/src/utils/constants.ts#L24).